package.json

```diff
@@ -151,7 +151,7 @@
       "architectures": [
         "x86"
       ],
-      "installTestPath": "./.omnisharp/OmniSharp.exe",
+      "installTestPath": "./.omnisharp/1.30.1/OmniSharp.exe",
```

Comment:
There are 21 instances of the string "1.30.1" in this file. We need a better approach. Thoughts?